lib/rentlinx/models/base.rb

```diff
@@ -41,15 +41,23 @@ def to_hash
     end
 
     def valid?
-      required_attributes.all? do |at|
-        !send(at).nil? && send(at) != ''
-      end
+      required_attributes.all? { |at| !blank?(send(at)) }
```

Comment:
Would `required_attributes.none? { |at| blank?(send(at)) }` be a bit more clear?